backend/src/test/java/de/neuefische/backend/controller/EventControllerTest.java

```diff
@@ -212,4 +212,368 @@ void deleteEvent() throws Exception {
+                                        "start": "2023-06-14T10:00:00Z",
+                                        "end": "2023-06-14T12:00:00Z",
+                                        "location": "Location 3",
+                                        "category": "MUSIC",
+                                        "creator": "Creator2",
+                                        "status": "DECLINED"
+                                }
+                                """)
+                        .with(csrf()))
+                .andExpect(status().is(201));
+
+        mockMvc.perform(MockMvcRequestBuilders.post("/api/event")
+                        .contentType("application/json")
+                        .content("""
+                                {
+                                        "title": "Title 4",
+                                        "description": "Description",
+                                        "start": "2023-06-14T10:00:00Z",
+                                        "end": "2023-06-14T12:00:00Z",
+                                        "location": "Location 4",
+                                        "category": "THEATRE",
+                                        "creator": "Creator1",
+                                        "status": "NEW"
+                                }
+                                """)
+                        .with(csrf()))
+                .andExpect(status().is(201));
+
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/event/status/NEW"))
+                .andExpect(status().isOk())
+                .andExpect(content().json("""
+                        [
+                             {
+                                        "title": "Title 1",
+                                        "description": "Description",
+                                        "start": "2023-06-14T10:00:00Z",
+                                        "end": "2023-06-14T12:00:00Z",
+                                        "location": "Location 1",
+                                        "category": "MUSIC",
+                                        "creator": "Creator1",
+                                        "status": "NEW"
+                             },
+                             {
+                                        "title": "Title 4",
+                                        "description": "Description",
+                                        "start": "2023-06-14T10:00:00Z",
+                                        "end": "2023-06-14T12:00:00Z",
+                                        "location": "Location 4",
+                                        "category": "THEATRE",
+                                        "creator": "Creator1",
+                                        "status": "NEW"
+                             }
+                         ]
+                        """));
+
+        mockMvc.perform(MockMvcRequestBuilders.get("/api/event/status/APPROVED"))
```

Comment:
Why test for different status messages? This test is just repeating itself with not much knowledge to gain.

Reply:
At this point I wanted to make sure that both a single object with a status and multiple objects can be returned. And since there are only 3 status values in total, I added the third one just for "completeness", even though it could have been omitted.